decorator.rb

```diff
@@ -0,0 +1,13 @@
+require './nameable'
+class Decorator < Nameable
+  attr_accessor :component
```

Comment:
- [ ] I think you meant `:nameable` instead of `:component` 😄 Please fix this issue. 🙏

Reply:
Oh, thank you for the heads up.